src/app/external-log-in/external-log-in/external-log-in.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <h4>{{ 'external-login.confirmation.header' | translate}}</h4>
+</div>
+<div class="row justify-content-center">
+  <ng-container *ngComponentOutlet="getExternalLoginConfirmationType(); injector: objectInjector;">
+  </ng-container>
+</div>
+<ds-alert class="row mt-2" [type]="AlertTypeEnum.Info" [attr.data-test]="'info-text'">
+  {{ informationText }}
+</ds-alert>
+<div class="row justify-content-center">
+  <div class="col-4 d-flex justify-content-end align-items-center">
+    <ng-container *ngIf="registrationData.email; else provideEmail">
+      <ds-confirm-email [registrationData]="registrationData" [token]="token"></ds-confirm-email>
+    </ng-container>
+    <ng-template #provideEmail>
+      <ds-provide-email [registrationId]="registrationData.id" [token]="token"></ds-provide-email>
+    </ng-template>
+  </div>
+  <div class="col-1 align-items-center d-flex justify-content-center">
+    <h4 class="mt-2">or</h4>
+  </div>
+  <div class="col-4 align-items-center d-flex justify-content-start">
+    <button class="btn block btn-lg btn-primary" (click)="openLoginModal(loginModal)">
+      {{'external-login.connect-to-existing-account.label' | translate}}
+    </button>
+  </div>
+</div>
+
+<ng-template #loginModal let-c="close" let-d="dismiss">
+  <div class="modal-header">
+    <h4 class="modal-title text-info"> {{'external-login.connect-to-existing-account.label' | translate}}</h4>
+  </div>
+  <div class="modal-body">
+    <div class="row justify-content-center">
+      <ds-log-in
+        [excludedAuthMethod]="relatedAuthMethod"
+        [showRegisterLink]="false"
+        [isStandalonePage]="true"
+      ></ds-log-in>
+    </div>
+  </div>
+  <div class="modal-footer">
+    <button type="button" class="btn btn-outline-primary btn-sm" (click)="c('Close click');clearRedirectUrl()">
```

Comment:
What is this "Close click"?  Is this accidental or maybe just for testing purposes?

Reply:
Is sent to the method `ModalRef#close(result?:any)` of the modal. Has been set like that just to register the event.